doc/basics.md

```diff
@@ -44,6 +48,31 @@ Note:
 - Visual Studio users should compile with the `_CRT_SECURE_NO_WARNINGS` flag to avoid warnings with respect to our use of standard C functions such as `fopen`.
 
 
+
+
+Using simdjson as a CMake dependency
+------------------
+
+
+
+You can include the  simdjson repository as a folder in your CMake project. In the parent
+`CMakeLists.txt` include the following lines:
+
+```
+set(SIMDJSON_JUST_LIBRARY ON CACHE STRING "Build just the library, nothing else." FORCE)
+add_subdirectory(simdjson EXCLUDE_FROM_ALL)
+```
+
+Elsewhere in your project, you can  declare dependencies on simdjson with lines such as these:
+
+```
+add_executable(myprogram myprogram.cpp)
+
```

Comment:
Don't think this newline helps either, and it might actually affect readability a tiny bit by making users think the pause matters.

Reply:
Done.